test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7.lua

```diff
@@ -11,11 +11,13 @@ aquad.OptimizeForPolarSymmetry(pquad, 4.0 * math.pi)
 
 -- Solver
 if string.find(k_method, "scdsa") or string.find(k_method, "smm") then
-  inner_linear_method = "classic_richardson"
+  inner_linear_method = "petsc_richardson"
```

Comment:
Did classic Richardson not work with CBC?

Reply:
It should work with both I would imagine (as you saw). I will revert this back to classic Richardson tonight or tomorrow morning. As the methods are coded up, they allow for multi-sweep. The single sweep is just enforced in the input file. I think warnings should probably be printed for AAH/single-sweep settings.